src/query.rs

```diff
@@ -152,6 +152,10 @@ impl Query {
 
         let mut builder = Request::builder().method(method).uri(url.as_str());
 
+        for (name, value) in &self.client.headers {
```

Comment:
What about inserts?

Reply:
There is a change in the `insert.rs` file to add the headers. Is this what you mean?